### PR TITLE
[TRANSLATION] translate Git.md

### PR DESCRIPTION
**Pull request by LingkKang**

### CODE REVIEW

**Comment by PKUFlyingPig** [comment]
下次可以新开一个分支用于提交 PR，以免 Git history 里会留有上次 PR 的信息。

File: docs/必学工具/Git.en.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+
+But trust me, like Vim, Git is a powerful tool that you will finally exclaim 'it's worth it!' after you mastered it. 
+
+## How to learn to use Git 
+
+Different from Vim, I don't suggest beginners use Git rashly without sufficient understanding of it. That's because it's inner logic can not be acquainted by practicing, but takes time to figure out. Here is my recommended learning path: 
+
+1. Read this [Git tutorial](https://missing.csail.mit.edu/2020/version-control/), and this video [Git tutorial (by 尚硅谷)](https://www.bilibili.com/video/BV1vy4y1s7k6) in Chinese. 
+2. Read Chap1 - Chap5 of this open source book [Pro Git](https://git-scm.com/book/en/v2). Yes, to learn Git, you need to read a book. 
+3. Now that you have understood its principles and most of its usages, it's time to consolidate those commands by practicing. It's also a philosophy that how to use Git properly. I personally think this blog [How to Write a Git Commit Message](https://chris.beams.io/posts/git-commit/) is really wroth reading.
```
Comment (PKUFlyingPig) [inline docs/必学工具/Git.en.md:17]: How to use Git properly is a kind of philosophy. I recommend reading this blog [How to Write a Git Commit Message](https://chris.beams.io/posts/git-commit/).

File: docs/必学工具/Git.en.md [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+
+## How to learn to use Git 
+
+Different from Vim, I don't suggest beginners use Git rashly without sufficient understanding of it. That's because it's inner logic can not be acquainted by practicing, but takes time to figure out. Here is my recommended learning path: 
+
+1. Read this [Git tutorial](https://missing.csail.mit.edu/2020/version-control/), and this video [Git tutorial (by 尚硅谷)](https://www.bilibili.com/video/BV1vy4y1s7k6) in Chinese. 
+2. Read Chap1 - Chap5 of this open source book [Pro Git](https://git-scm.com/book/en/v2). Yes, to learn Git, you need to read a book. 
+3. Now that you have understood its principles and most of its usages, it's time to consolidate those commands by practicing. It's also a philosophy that how to use Git properly. I personally think this blog [How to Write a Git Commit Message](https://chris.beams.io/posts/git-commit/) is really wroth reading. 
+4. You are now in love with Git, and you are not content with managing it, you want to build it by yourself! Great, that's exactly what I was thinking. [This tutorial](https://wyag.thb.lt/) may satisfy you!
```
Comment (PKUFlyingPig) [inline docs/必学工具/Git.en.md:18]: You are now in love with Git and are not content with only using it, you want to build a Git by yourself! Great, that's exactly what I was thinking. [This tutorial](https://wyag.thb.lt/) will satisfy you!

File: docs/必学工具/Git.en.md [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+2. Read Chap1 - Chap5 of this open source book [Pro Git](https://git-scm.com/book/en/v2). Yes, to learn Git, you need to read a book. 
+3. Now that you have understood its principles and most of its usages, it's time to consolidate those commands by practicing. It's also a philosophy that how to use Git properly. I personally think this blog [How to Write a Git Commit Message](https://chris.beams.io/posts/git-commit/) is really wroth reading. 
+4. You are now in love with Git, and you are not content with managing it, you want to build it by yourself! Great, that's exactly what I was thinking. [This tutorial](https://wyag.thb.lt/) may satisfy you! 
+5. What? Building your own Git is not enough? You're quite promising! I also like reinventing the wheel. This two GitHub projects, [build-your-own-x](https://github.com/danistefanovic/build-your-own-x) and [project-based-learning](https://github.com/tuvtran/project-based-learning), include any wheel-reinventing tutorial you can imagine, i.e., to build your own text editor, virtual machine, docker, TCP and so on.
```
Comment (PKUFlyingPig) [inline docs/必学工具/Git.en.md:19]: What? Building your own Git is not enough? Seems that you are also passionate about reinventing the wheels. These two GitHub projects, [build-your-own-x](https://github.com/danistefanovic/build-your-own-x) and [project-based-learning](https://github.com/tuvtran/project-based-learning), collected many wheel-reinventing tutorials, e.g., text editor, virtual machine, docker, TCP and so on.

**Comment by LingkKang** [comment]
> 下次可以新开一个分支用于提交 PR，以免 Git history 里会留有上次 PR 的信息。

好的

**Pull request merged**